Logo/Logo/Views/LogoTwoView.swift

```diff
@@ -0,0 +1,210 @@
+//
+//  LogoTwoView.swift
+//  Logo
+//
+//  Created by Cora Jacobson on 7/27/20.
+//  Copyright © 2020 Cora Jacobson. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+class LogoTwoView: UIView {
+    
+    var name: String = "Dice"
+    
+    // MARK: - View Lifecycle
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = UIColor.clear
+        setNeedsDisplay()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        backgroundColor = UIColor.clear
+    }
```

Comment:
I know that your setup in all of these views is minimal, but still might be a nice idea to create a shared setUp or configure or whatever you want to call it method in case you wanted to add more set up code later.